README.md

```diff
@@ -19,5 +19,6 @@ Obviously Stylelint should be installed too.
 - [declarations-before-nesting](./docs/rules/declarations-before-nesting.md) — forces block rules be placed above the nested blocks.
 - [no-transition-all](./docs/rules/no-transition-all.md) — prevents use `all` value for `transition` & `transition-property` props.
 - [rgba-hex-color-only](./docs/rules/rgba-hex-color-only.md) — prevents use `rgba` function with non-hex color as a first argument.
+- [selector-max-type](./docs/rules/selector-max-type.md) — limit the number of type selectors in a selector (compatible with [bem-react-helper](https://github.com/igoradamenko/bem-react-helper)).
```

Comment:
🤔 

This rule is not related to bem-react-helper. The helper is `b()` func in React components. 

I think you wanted to note here our [scss-vars-loader](https://github.com/funbox/scss-vars-loader).

Reply:
Oh yes, of course. Thanks.